src/axom/inlet/Table.hpp

```diff
@@ -639,18 +663,56 @@ class Table : public Verifiable
     T result;
```

Comment:
I guess this assumes that `T` is default-constucible. 

This is almost definitely a reasonable assumption for inlet, but is an assumption nevertheless. This should probably be added to the doxygen for `get<>`  (assuming it's not already there)

Reply:
In this case the `T` is `unordered_map<int, UserDefinedType>` which is default-constructible even if `UserDefinedType` is not.  This will forward to the `FromInlet<UserDefinedType>` specialization which allows the user to construct the `UserDefinedType` instance however they see fit.

Reply:
Perhaps a doxy `\tparam` comment on `T` would be appropriate?  (Might already be there, it's difficult to navigate through github)